### PR TITLE
feat(Checkbox): add indeterminate state

### PR DESCRIPTION
**Pull request by AadneRo**
Visual changed for checkbox indeterminate state

- [x] fix undesired change of checked->unchecked animation
- [x] Add sbanken theme  
- [x] fix shanked error+hover state -> this is wrong compared to the designs

### CODE REVIEW

File: packages/dnb-eufemia/src/components/checkbox/__tests__/__image_snapshots__/checkbox-for-sbanken-checked-have-to-match-checkbox-in-indeterminate-state.snap.png
Comment (AadneRo) [inline packages/dnb-eufemia/src/components/checkbox/__tests__/__image_snapshots__/checkbox-for-sbanken-checked-have-to-match-checkbox-in-indeterminate-state.snap.png:1]: these are the sbanken screenshots, need to double check these with designer

File: packages/dnb-eufemia/src/components/skeleton/__tests__/__image_snapshots__/skeleton-for-sbanken-have-to-match-all-components-horizontal.snap.png
Comment (tujoworker) [inline packages/dnb-eufemia/src/components/skeleton/__tests__/__image_snapshots__/skeleton-for-sbanken-have-to-match-all-components-horizontal.snap.png:1]: The skeleton of the checkbox looks like to not be set to the correct color. We even may just remove it.

File: packages/dnb-eufemia/src/components/skeleton/__tests__/__image_snapshots__/skeleton-for-sbanken-have-to-match-all-components-vertical.snap.png
Comment (tujoworker) [inline packages/dnb-eufemia/src/components/skeleton/__tests__/__image_snapshots__/skeleton-for-sbanken-have-to-match-all-components-vertical.snap.png:1]: Here as well.

**Review by tujoworker — APPROVED** [review]
Very great work ⭐

File: packages/dnb-eufemia/src/components/skeleton/__tests__/__image_snapshots__/skeleton-for-sbanken-have-to-match-all-components-horizontal.snap.png
Comment (tujoworker) [reply packages/dnb-eufemia/src/components/skeleton/__tests__/__image_snapshots__/skeleton-for-sbanken-have-to-match-all-components-horizontal.snap.png:1]: We will fix it in an upcoming PR 👍

**Pull request merged**